### PR TITLE
`plot` updates (`detach` mode, `v_common` arg)

### PR DESCRIPTION
**Pull request by HollowPrincess**
Updates in the `Plot`:
 - Add detach modes (`True`, `False`, `'save'`) for running in daemon process
 - Add `v_common` argument which affects `vmin` and `vmax` plotting arguments

### CODE REVIEW

File: batchflow/plotter/plot.py [hunk 1]
```diff
@@ -23,6 +25,29 @@
 
 
 
+# Decorators
+def run_in_daemon_process(func):
```
Comment (SergeyTsimfer) [inline batchflow/plotter/plot.py:29]: `detachable`?
`maybe_detach`?

File: batchflow/plotter/plot.py [hunk 2]
```diff
@@ -185,6 +209,28 @@ def update(self, data):
         if self.mode == 'loss':
             raise NotImplementedError("Updating layer data is not in supported in 'loss' mode. ")
 
+    def _parse_v_ranges(self, data):
+        """ Parse vmin and vmax values from the `self.config`. """
+        vmin = self.config.get('vmin', np.nanmin(data))
+        vmax = self.config.get('vmax', np.nanmax(data))
+        v_common = self.config.get('v_common', None)
```
Comment (SergeyTsimfer) [inline batchflow/plotter/plot.py:216]: Do you have any other suggestions for naming of this parameters? I am not entirely sure about `v_common`, but don't have better alternatives:(

`vabs`
`vminmax`

File: batchflow/plotter/plot.py [hunk 3]
```diff
@@ -1543,7 +1614,8 @@ def plot(self, data=None, combine='overlay', mode='image', show=True, force_show
             self.close()
 
         if save or 'savepath' in self.config:
-            self.save()
+            detach_save = detach=='save'
```
Comment (SergeyTsimfer) [inline batchflow/plotter/plot.py:1617]: I would rather have a separate parameter `detach_save`

**Review by SergeyTsimfer — APPROVED** [review]

File: batchflow/plotter/plot.py (hunk 3 above)
Comment (HollowPrincess) [reply batchflow/plotter/plot.py:1617]: No

File: batchflow/plotter/plot.py [hunk 4]
```diff
@@ -883,6 +941,11 @@ class Plot:
     fix_config : bool
         If False, every time `plot` is called update config with provided keyword arguments, replacing older parameters.
         If True, fix plotter config as provided on initialization. Usefull, if one want to reuse this config on updates.
+    detach : {True, False, 'save'}
```
Comment (AlexeyKozhevin) [inline batchflow/plotter/plot.py:944]: Add default value description

**Review by SergeyTsimfer — APPROVED** [review]

**Pull request merged**